Include stacktrace flag (#6267)
keystonejs/keystone: .changeset/real-sheep-end.md

---
'@keystone-next/keystone': minor
'@keystone-next/types': minor
---

> Add a `config.graphql.debug` option, which can be used to control with debug information such as stack traces are include in the errors returned by the GraphQL API.

Reviewer: ```suggestion
Added `config.graphql.debug` option, which can be used to control whether debug information such as stack traces are included in the errors returned by the GraphQL API.
```
